### DISCUSSION TITLE
Outreachy 2023 Applicants

### DISCUSSION THREAD

**Discussion by CeciliaHinga** [ushahidi/platform#4636] [Outreachy and Google Summer of Code]
**Welcome to Ushahidi and the application-process for Outreachy!**

We are so excited you are here!! 🌻 🎉 💐

There are a few steps you need to complete before starting working on an issue. These are:

1. Read through our [Code of Conduct](https://docs.ushahidi.com/platform-developer-documentation/code-of-conduct)
2. Get set up and say hello in [Gitter](https://gitter.im/ushahidi/gsoc-outreachy) to communicate, help and get help from the community.
3. Get your [developer-environment](https://docs.ushahidi.com/platform-developer-documentation/development-and-code/setup_alternatives) set up. We most often recommend using the Vagrant-setup, but XAMPP usually works fine too. Once you have set up your API, remember to setup the Platform-client as well! If you get stuck, reach out to the Gitter-channel to get help.
4. Explore the Ushahidi Platform and the different concepts like surveys (also called "forms" in some places) and posts. Try setting up a survey, a few categories, and add some posts. The user-documentation is found here: https://docs.ushahidi.com/ushahidi-platform-user-manual/
5. Once you have completed steps 1-4, send a reply to this discussion with screenshots of your setup in the browser and select an issue to work with.

**To keep in mind when starting working on issues:**

Good issues to work with are found here:

- [Outreachy2023](https://github.com/ushahidi/platform-client-mzima/issues?q=is%3Aissue+is%3Aopen+label%3Ae2e-test-automation)
- You can only work on one issue at a time. We are looking for the quality of the contributions rather than the number of contributions when evaluating the applications.
- Once you are assigned to an issue, we want you to submit a pull request within 3 days, even if it is not yet complete or a working solution. This is to make our contributors work "in the open" as is often done in Open Source, and make sure no issues are not being worked on. If you don't show progress within 3 days, we will open it up for other students. GitHub has a good guide on how to submit pull requests from a fork: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork

**Remember**
Ushahidi is an open-source community. The mentors are trying their best to reply to your questions, but there may be some delays. By helping other applicants out in Gitter, you are also helping the mentors. This is recognized by us and seen as something very positive in evaluating the applications. So if you see someone asking for help in the chat and know the answer, please help them out! Keep the tone nice and collaborative.

**Privacy info/personal info**
During the application-process, Ushahidi will store your Github-handle in a google-sheet. This is done only to make the application process as smooth as possible and to keep track of which issues are being worked on and the progress of each issue. The information stored will be deleted once the application-process is over.

**Comment by oyenuga17** [comment]
<img width="1533" alt="Screenshot 2023-10-02 at 19 03 36" src="https://github.com/ushahidi/platform/assets/64274826/a33c5921-97d9-4323-a3f2-bb5a3ff74db2">
<img width="1536" alt="Screenshot 2023-10-02 at 19 09 50" src="https://github.com/ushahidi/platform/assets/64274826/76983408-fd0e-4090-a1ea-8c98cf97b989">
- I have successfully set up the project on my system and I am taking a look at the good first issues to see what valuable contributions i can add to it .. while i wait on **outreachy 2023** issues  as nothing is currently [here](https://github.com/ushahidi/platform/labels/Outreachy%2023)

@CeciliaHinga

**Comment by CeciliaHinga** [comment]
@AmTryingMyBest

**Comment by shakirandagire** [comment]
I have finally been able to setup the project.
<img width="1512" alt="Screenshot 2023-10-06 at 14 13 52" src="https://github.com/ushahidi/platform/assets/6957960/f6c9dc95-5685-49b2-a41c-155d349c32a0">
<img width="1512" alt="Screenshot 2023-10-06 at 14 25 06" src="https://github.com/ushahidi/platform/assets/6957960/698b5996-6a21-4c62-8065-a414f70376c9">

**Comment by Caleno-R** [comment]
I am thrilled to have set up the project! Now, I proceed to contribute to Ushahidi!

<img width="1704" alt="image" src="https://github.com/ushahidi/platform/assets/59660721/c758a8fd-d087-4f4d-8b7c-69335f91f778">

<img width="1704" alt="image" src="https://github.com/ushahidi/platform/assets/59660721/94ce3403-7ba0-42b0-80dd-d0d6463423bc">

**Comment by WanjaNjunge** [comment]
Finally able to set everything up!

![Screenshot from 2023-10-16 07-53-10](https://github.com/ushahidi/platform/assets/114565394/38d2cd97-7800-40d0-a01d-9031e5172952)
![Screenshot from 2023-10-16 16-11-09](https://github.com/ushahidi/platform/assets/114565394/67dc3659-46f1-4711-951a-d4cdfc799a1e)
